db_management/questionnairedb/data/test/omi/patients.sql

```diff
@@ -2,4 +2,25 @@
 --
 -- SPDX-License-Identifier: AGPL-3.0-or-later
 
--- patients get synced automatically via the DB event that runs periodically
+-- Legacy (deleted) test patients are kept until all their remaining data in QuestionnaireDB is cleaned up
+-- Otherwise, data linked to certain patientIds will be orphaned
+REPLACE INTO `patient` (`ID`, `hospitalId`, `externalId`, `deleted`, `deletedBy`, `creationDate`, `createdBy`, `lastUpdated`, `updatedBy`) VALUES
+(1, -1, 51, 1, '', '2025-01-01 00:00:00', 'TEST_DATA_MARGE_DELETED', '2025-01-01 00:00:00', 'TEST_DATA'),
+(2, -1, 52, 1, '', '2025-01-01 00:00:00', 'TEST_DATA_HOMER_DELETED', '2025-01-01 00:00:00', 'TEST_DATA'),
+(3, -1, 53, 1, '', '2025-01-01 00:00:00', 'TEST_DATA_BART_DELETED', '2025-01-01 00:00:00', 'TEST_DATA'),
+(5, -1, 55, 1, '', '2025-01-01 00:00:00', 'TEST_DATA_MONA_DELETED', '2025-01-01 00:00:00', 'TEST_DATA'),
+(6, -1, 56, 1, '', '2025-01-01 00:00:00', 'TEST_DATA_FRED_DELETED', '2025-01-01 00:00:00', 'TEST_DATA'),
+(7, -1, 57, 1, '', '2025-01-01 00:00:00', 'TEST_DATA_PEBBLES_DELETED', '2025-01-01 00:00:00', 'TEST_DATA'),
+(8, -1, 58, 1, '', '2025-01-01 00:00:00', 'TEST_DATA_WEDNESDAY_DELETED', '2025-01-01 00:00:00', 'TEST_DATA'),
+
+-- Laurie and Rory's patient rows are in reverse order in this database, as opposed to OpalDB.Patient
+(9, -1, 92, 0, '', '2025-01-01 00:00:00', 'TEST_DATA_LAURIE', '2025-01-01 00:00:00', 'TEST_DATA'),
+(10, -1, 59, 0, '', '2025-01-01 00:00:00', 'TEST_DATA_RORY', '2025-01-01 00:00:00', 'TEST_DATA');
+
+-- Update all timestamps to the current time
+UPDATE `patient`
+SET
+`creationDate` = now(),
+`lastUpdated` = now();
+
+-- All remaining patients are synced in order, automatically, via the DB event SyncPublishQuestionnaire that runs periodically
```

Comment:
@mschoettle This is still pretty fragile, if we decide to add a new test patient in the middle of the list in OpalDB (e.g. PatientSerNum 54), the synced list might get offset again and break references to `patient.ID`.

What do you think about adding all the rows we need back here?

Reply:
Maybe we insert all test patients then in line with #578?

Reply:
Yeah, I think inserting all of them might be best.

Reply:
Ok, let's add them all then. I am doing the same for legacy patients in #578.